### ISSUE TITLE
Has to close and reopen game in order to use files added to the the assets folder.

### ISSUE THREAD

**Issue by lynxieles** [chrismaltby/gb-studio#507]
**Describe the bug**
Refreshing assets doesn't allow them to appear in the sprite or background selector without fully closing and relaunching the game.

**To Reproduce**
1. Open the assets tab via the button in the top right of the screen.
2. Add a new asset via copy and paste or drag and drop.
3. go to use the new asset. find it's not there.
4. save the game and relaunch the app. 
5. can now use the asset.

**Expected behavior**
In 1.2.1 I could access the assets added to the folder as soon as I clicked "Reload Assets"

**Screenshots**
Added a video in zip file because a photo would not be helpful
The name of the file added was SCP 682.
<img width="1239" alt="Screen Shot 2020-08-11 at 2 23 08 PM" src="https://user-images.githubusercontent.com/68176808/91671902-bd26c100-eae7-11ea-902f-722e0e71fca5.png">

**Platform (please complete the following information):**
 - OS: Mac 10.15.4
 - App version 2.0 beta

**Additional context**

**Comment by um3k** [comment]
I am experiencing the same problem in Windows 10, so it's not platform specific.

**Comment by RichardULZ** [comment]
This is a difficult problem, as every time it seems fixed, another issue is reported. Many users had issues with bad/buggy reloading in 1.2.1, so to hear it is still in 2.0, possibly in a different way, is concerning.

I can not find the zipped video you mentioned.

**Comment by lynxieles** [comment]
Yeah it seems to have replaced the file. let me try again.
[Untitled.mov.zip](https://github.com/chrismaltby/gb-studio/files/5147798/Untitled.mov.zip)
